### PR TITLE
install guide

### PR DESCRIPTION
**Pull request by alexzielenski**
Adds `Install.md` with some information about easy and manual installation.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+After running the final `kubectl apply` the CEL Admission Webhook should
+be active on your kind cluster and ready to enforce policies.
+
+
+# Manual Installation
+
+> 🚧 WORK IN PROGRESS 🚧
+
+Requirements:
+  1. Kubernetes 1.x+ (TODO: determine minimum k8s version for webhook)
```
Comment (cici37) [inline INSTALL.md]: Webhook is beta since 1.9 and become stable in 1.16

Comment (cici37) [reply INSTALL.md]: Since CRD is stable in 1.16 as well, how about to set the requirement to 1.16?

File: INSTALL.md
Comment (cici37) [inline INSTALL.md:1]: Let's mention the install.md inside README as well? Maybe together with the motivation of this repo in README. It could be a separate PR as well^^

File: INSTALL.md (hunk 1 above)
Comment (alexzielenski) [reply INSTALL.md]: added

File: INSTALL.md
Comment (alexzielenski) [reply INSTALL.md:1]: README is currently pretty bare and needs lots of love. We can add link when populating that. Think separate PR better for now so we can get these merged in
